addons/web_editor/static/lib/odoo-editor/odoo-editor.js

```diff
@@ -4072,7 +4072,9 @@ var exportVariable = (function (exports) {
                     }
                 }
             }
-            this.dispatchEvent(new Event('observerApply'));
+            if (records.length) {
+                this.dispatchEvent(new Event('observerApply'));
+            }
```

Comment:
Not sure if we can fix bugs directly inside the odoo-editor lib or if this must be fixed in another repo first. Did you check with the editor team about this fix? If not, can you check with them if they are ok with the fix and if it must be fixed somewhere else? Thanks :+1: 

Also: your commit message might want to explain what functional aspect this actually fixes instead of just explaining the code :wink:

Reply:
I opened a PR for that here https://github.com/odoo-dev/odoo-editor/pull/244